palindrom-client.html

```diff
@@ -485,22 +483,9 @@
                         }
                     }
                 }
-            },
-            validateJSONPatch() {
-                console.info('inherit validateJSONPatch from fast-json-patch');
-            },
-            observeJSONPatch(obj, callback) {
-                this.palindromObserver = callback;
-                return callback;
-            },
-            unobserveJSONPatch() {
-                // FIXME this is dummy as hell
-                this.palindromObserver = noop;
```

Comment:
Were these three methods completely unused and undocumented?

Reply:
Exactly. I don't even know what they do. I searched the repo and all Github and couldn't find an instance of `observeJSONPatch` or `unobserveJSONPatch` except with `palindrom-client.html` files.